src/SourceBrowser/src/Directory.Packages.props

```diff
@@ -41,6 +41,7 @@
     <PackageVersion Include="Shouldly" Version="4.3.0" />
 
     <!-- other dependencies -->
+    <PackageVersion Include="Basic.CompilerLog.Util" Version="0.9.17" />

```

Comment:
Can you please follow up and make sure https://www.nuget.org/packages/Basic.CompilerLog.Util/ gets a license and source repository added to it's NuGet package?  You might even get a component governance alert from this after merging into official build.

Reply:
Hm, I can, but I believe this is already used in other repos, e.g., [in sdk](https://github.com/dotnet/sdk/blob/4b4953ef1c4a999dcb3e6036fe3ecc9e3a87390d/Directory.Packages.props#L8) (although perhaps only in tests there).

Reply:
Is it fine to follow up separately or is this blocking merge of the PR?